libbpf-cargo/src/test.rs

```diff
@@ -702,7 +684,7 @@ fn test_skeleton_builder_basic() {
     let skel = NamedTempFile::new().unwrap();
     SkeletonBuilder::new(proj_dir.join("src/bpf/prog.bpf.c"))
         .debug(true)
-        .clang("/bin/clang")
+        .clang("clang")
```

Comment:
this should probably be a default value and shouldn't be required to be specified explicitly

Reply:
Yeah, it should. I'll change it

Reply:
Maybe drop `.clang("clang")` in one of the cases to make sure that still works?